HopsanGUI/Widgets/HcomWidget.cpp

```diff
@@ -551,6 +595,30 @@ void TerminalConsole::keyPressEvent(QKeyEvent *event)
     }
 }
 
+//! @brief Inserts a selected completion from autocompleter
+void TerminalConsole::insertCompletion(const QString& completion)
+{
+    QString text = completion;
```

Comment:
If the first thing we do is to take a copy of the const reference "completion", and then never use it again. We could instead let the argument be a non-const Qstring, not a big deal but removes the need for an extra text variable.